torch/csrc/autograd/forward_grad.h

```diff
@@ -84,17 +84,18 @@ struct ForwardGrad;
 #define EXPECTED_MAX_LEVEL 2
 
 struct TORCH_API ForwardADLevel {
-    ForwardADLevel(uint64_t idx): idx_(idx) {}
-    ~ForwardADLevel();
-
-    static uint64_t get_next_idx();
-    static void release_idx(uint64_t idx);
-    static std::shared_ptr<ForwardADLevel> get_by_idx(uint64_t idx);
-    static std::shared_ptr<ForwardADLevel> try_get_by_idx(uint64_t idx);
-
-    void erase(const std::shared_ptr<ForwardGrad>& grad) {
-        std::lock_guard<std::mutex> lock(mutex_);
-        grads_.erase(grad);
+  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
```

Comment:
the formatting seems to have been messed up here (not that it really matters, we can auto clang format this), but was that the result of the script inserting the `NOLINTNEXTLINE`s?

Reply:
Yes, the script inserted those. I'll auto clang format this